src/models/room-state.ts

```diff
@@ -419,13 +428,26 @@ export class RoomState extends TypedEventEmitter<EmittedEvents, EventHandlerMap>
                 this.setBeacon(event);
             }
 
-            const lastStateEvent = this.getStateEventMatching(event);
+            const lastStateEv = this.getStateEventMatching(event);
```

Comment:
Why did this get renamed to something less clear?

Reply:
Otherwise, we had a lot more linewraps that made I judged as harder to read overall.

Reply:
That's only because on L435 and L436 you're using the array syntax to set variables which is really redundant. It'd be clearer as 4 separate instantiations.